email.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const nodemailer = require('nodemailer');
+
+/**
+* Sends email message
+* @param {Object} mailOpts has keys `from`, `to`, `subject`, `text`
+* @param {Object} smtpConfig has keys `host` and `port`
+* @return {Promise} resolves to 'success' if email is sent
```

Comment:
Is the string `'success'` significant?

An alternative to passing back a string is "As long as the Promise resolves (e.g., not reject) the operation was a success"